mods/tuxemon/maps/timber_town.tmx

```diff
@@ -155,10 +155,25 @@
    </properties>
   </object>
   <object id="181" name="Player Spawn" type="event" x="560" y="48" width="16" height="16"/>
+  <object id="188" name="T oRoute 7 Also" type="event" x="480.766" y="623.937" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport witcher_route_7.tmx,8,3"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing down"/>
+   </properties>
+  </object>
+  <object id="189" name="To_Route_7" type="event" x="464.46" y="623.352" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport witcher_route_7.tmx,8,3"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing down"/>
+   </properties>
+  </object>
+  <object id="190" x="404.801" y="388.55"/>
  </objectgroup>
  <objectgroup color="#ff0000" id="6" name="Collision">
   <object id="119" type="collision" x="0" y="0" width="32" height="464"/>
-  <object id="120" type="collision" x="0" y="608" width="640" height="32"/>
+  <object id="120" type="collision" x="0" y="608" width="447.202" height="32"/>
```

Comment:
do not use fractional values.  use the CTRL key to snap the edges to the tile grid